bbot/modules/paramminer_headers.py

```diff
@@ -174,7 +174,7 @@ async def handle_event(self, event):
                 self.debug(f"Error initializing compare helper: {e}")
                 return
             batch_size = await self.count_test(url)
-            if batch_size == None or batch_size <= 0:
+            if batch_size is None or batch_size <= 0:
```

Comment:
Slightly better because it correctly deals with any falsey value of `batch_size`.
```suggestion
            if batch_size or 0 <= 0:
```